tfhe/src/core_crypto/algorithms/lwe_private_functional_packing_keyswitch.rs

```diff
@@ -162,6 +162,11 @@ pub fn par_private_functional_keyswitch_lwe_ciphertext_into_glwe_ciphertext_with
         "This operation currently only supports native moduli"
     );
 
+    assert!(
+        thread_count.0 != 0,
+        "Got thread_count == 0, this is not supported"
+    );
+
```

Comment:
Clippy did not complain about using `assert!` and not `assert_ne!` ? 😮

Reply:
probably because I added an error message